spec/tunnel/tunnel_spec.rb

```diff
@@ -41,8 +41,8 @@
       expect(Ngrok::Tunnel.addr).to eq(3001)
     end
 
-    it "has valid ngrok_url" do
-      expect(Ngrok::Tunnel.ngrok_url).to be =~ /http:\/\/.*ngrok\.io$/
+    it "has nil ngrok_url" do
+      expect(Ngrok::Tunnel.ngrok_url).to be nil
```

Comment:
what's the deal here? Does the new version not return the url?

Reply:
Ngrok v3 defaults to HTTPS only now

Reply:
I still think this method should return the https URL though?

Reply:
@rochers At the moment the http and https URLs are represented by ngrok_url and ngrok_url_https. Are you thinking they would both return the HTTPS one if that's the only one running?